website/docs/r/cloudwatch_event_connection.html.markdown

```diff
@@ -0,0 +1,200 @@
+
+`oauth` support the following:
+
+* `authorization_endpoint` - (Required) A username for the authorization.
+* `http_method` - (Required) A password for the authorization. Created and stored in AWS Secrets Manager.
+* `client_parameters` - (Required) Contains the client parameters for OAuth authorization. Contains the following two parameters.
+    * `client_id` - (Required) The client ID for the credentials to use for authorization. Created and stored in AWS Secrets Manager.
+    * `client_secret` - (Required) The client secret for the credentials to use for authorization. Created and stored in AWS Secrets Manager.
+* `oauth_http_parameters` - (Required) OAuth Http Parameters are additional credentials used to sign the request to the authorization endpoint to exchange the OAuth Client information for an access token. Secret values are stored and managed by AWS Secrets Manager. A maximum of 1 are allowed. Documented below.
+
+`invocation_http_parameters` and `oauth_http_parameters` support the following:
+
+* `body` - (Optional) Contains additional body string parameters for the connection. You can include up to 100 additional body string parameters per request. Each additional parameter counts towards the event payload size, which cannot exceed 64 KB. Each parameter can contain the following:
+    * `key` - (Required) The key for the parameter.
+    * `value` - (Required) The value associated with the key. Created and stored in AWS Secrets Manager if is secret.
+    * `is_value_secret` - (Optional) Specified whether the value is secret.
+
+* `header` - (Optional) Contains additional header parameters for the connection. You can include up to 100 additional body string parameters per request. Each additional parameter counts towards the event payload size, which cannot exceed 64 KB. Each parameter can contain the following:
+    * `key` - (Required) The key for the parameter.
+    * `value` - (Required) The value associated with the key. Created and stored in AWS Secrets Manager if is secret.
+    * `is_value_secret` - (Optional) Specified whether the value is secret.
+
+* `query_string` - (Optional) Contains additional query string parameters for the connection. You can include up to 100 additional body string parameters per request. Each additional parameter counts towards the event payload size, which cannot exceed 64 KB. Each parameter can contain the following:
+    * `key` - (Required) The key for the parameter.
+    * `value` - (Required) The value associated with the key. Created and stored in AWS Secrets Manager if is secret.
+    * `is_value_secret` - (Optional) Specified whether the value is secret.
+
+## Attributes Reference
+
+In addition to all arguments above, the following attributes are exported:
+
+* `arn` - The Amazon Resource Name (ARN) of the event bus.
+
+
+## Import
+
+EventBridge event buses can be imported using the `name`, e.g.
```

Comment:
This is a copy paste error from the event bus page.

Reply:
Great spot! Fixed :)